cloud/azure/modules/app/main.tf

```diff
@@ -110,7 +110,8 @@ resource "azurerm_app_service" "civiform_app" {
     DB_JDBC_STRING = "jdbc:postgresql://${local.postgres_private_link}:5432/postgres?ssl=true&sslmode=require"
 
     STORAGE_SERVICE_NAME = "azure-blob"
-    STAGING_HOSTNAME     = "sgtest-full-mammal.azurewebsites.net" // TODO(sgoldblatt): update this to staging when dns is set up
+    STAGING_HOSTNAME     = "sgtest-full-mammal.azurewebsites.net"                                      // TODO(sgoldblatt): update this to staging when dns is set up
```

Comment:
Nit: Is there a tracking bug we should link here?

Reply:
done!